ambry-router/src/main/java/com.github.ambry.router/GetBlobOperation.java

```diff
@@ -393,6 +421,10 @@ void completeRead() {
     private void updateChunkingAndSizeMetricsOnSuccessfulGet() {
       routerMetrics.getBlobSizeBytes.update(bytesWritten);
       routerMetrics.getBlobChunkCount.update(numChunksTotal);
+      if (hasRange) {
+        routerMetrics.getBlobWithRangeSizeBytes.update(bytesWritten);
```

Comment:
I added the latency metrics so that we can ensure that there aren't any significant performance regressions compared to normal gets.  The total time metric might be less useful because that is very dependent on how many bytes are requested.  I thought the range size could be useful to profile usage.